src/clickonce/launcher/Program.cs

```diff
@@ -150,6 +151,17 @@ private static void AddClickOnceEnvironmentVariables()
                 Environment.SetEnvironmentVariable("ClickOnce_UpdatedApplicationFullName", ad.UpdatedApplicationFullName?.ToString());
                 Environment.SetEnvironmentVariable("ClickOnce_UpdatedVersion", ad.UpdatedVersion?.ToString());
                 Environment.SetEnvironmentVariable("ClickOnce_UpdateLocation", ad.UpdateLocation?.ToString());
+
+                // ClickOnce ActivationData
+                string[] activationData = AppDomain.CurrentDomain?.SetupInformation?.ActivationArguments?.ActivationData;
+                if (activationData != null && activationData.Count() > 0)
```

Comment:
Wouldn't `Length` be better here since `Count` is a computed result?

Reply:
They should have equivalent value, in this case, so we could use either. I used `count` to match the env. var. name I'm using - `ClickOnce_ActivationData_Count`. It does require Linq and is slightly slower - but this array usually has zero or only 1 element.

Reply:
Pushed the change - removing Linq dependency and using `Length`.

Reply:
So these environment variables will get inherited all the way through to the application process where the app can choose to read these variables?

Reply:
Yes, that is correct. It is the model we've been using for passing data from Launcher (which is a ClickOnce-activated app and has access to additional data like `ApplicationDeployment.CurrentDeployment`) to the .NET (Core) app being launched.